packages/cli/src/lib/setup/setupBackup.ts

```diff
@@ -219,18 +219,6 @@ export class BackupRestore {
      * @param name - backup name
      */
     private _packBackup(name: string): Promise<string> {
-        // 2021_10_25 BF (TODO): store letsencrypt files too
```

Comment:
hm ...aber dann müsste setup first das verzeichnis wegräumen oder? weil sonst landet es plötzlich imbackup

Reply:
I think we already discussed in Telegram that this does not happen